### PR TITLE
fix: Update Apache 2 license to make it discoverable by github

### LINKED ISSUE #18: LICENSE is not discovered in terraform-aws-modules repos
Opening this issue here. It comes from https://github.com/terraform-aws-modules/terraform-aws-eks/issues/1068.

### PR DESCRIPTION
**Pull request by barryib**
## Description

Make the LICENSE discoverable by Github.

I added this license from the Github license template.

## Motivation and Context

Resolves https://github.com/terraform-aws-modules/meta/issues/18

### CODE REVIEW

File: files/terraform-modules/LICENSE [hunk 1]
```diff
@@ -1,13 +1,201 @@
...
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
```
Comment (daroga0002) [inline files/terraform-modules/LICENSE]: it works without this appendix.

File: LICENSE [hunk 2]
```diff
@@ -1,11 +1,201 @@
...
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
```
Comment (daroga0002) [inline LICENSE]: it works without this appendix.

File: files/terraform-modules/LICENSE (hunk 1 above)
Comment (barryib) [reply files/terraform-modules/LICENSE]: Right. fixed in 8f75a13

File: LICENSE (hunk 2 above)
Comment (barryib) [reply LICENSE]: Right. fixed in 8f75a13

**Review by daroga0002 — APPROVED** [review]
LGTM

**Pull request closed without merging**